test/unit/storage/ExtensionBasedMapper.test.ts

```diff
@@ -0,0 +1,18 @@
+import { RuntimeConfig } from '../../../src/init/RuntimeConfig';
+import { ExtensionBasedMapper } from '../../../src/storage/ExtensionBasedMapper';
+
+describe('A ExtensionBasedMapper', (): void => {
+  const base = 'http://test.com/';
+  const rootFilepath = 'uploads/';
+  const resourceMapper = new ExtensionBasedMapper(new RuntimeConfig({ base, rootFilepath }));
+
+  it('Returns correct url.', async(): Promise<void> => {
```

Comment:
The complete test name is equal to the concatenation of all nested blocks, so in this case "A ExtensionBasedMapper Returns correct url.". So what I'm trying to say is that these should start lower case 😉 . (Also should be "An ExtensionBasedMapper).